android/src/main/java/com/bitmovin/player/reactnative/DrmModule.kt

```diff
@@ -75,9 +75,9 @@ class DrmModule(private val context: ReactApplicationContext) : ReactContextBase
      */
     @ReactMethod
     fun initWithConfig(nativeId: NativeId, config: ReadableMap?) {
-        uiManager()?.addUIBlock {
-            if (!drmConfigs.containsKey(nativeId) && config != null) {
-                JsonConverter.toWidevineConfig(config)?.let {
+        context.uiManagerModule?.addUIBlock {
```

Comment:
Why isn't this class a BitmovinBaseModule with the same resolveOnUIThread logic?

Reply:
The DRM module was not refactor. It is now (by many patches.)